packages/webapp/src/webview/i18n/i18n.json

```diff
@@ -33,5 +33,6 @@
     "SHARE_THIS_GUIDE": "Share this guide",
     "COPY_THIS_LINK": "Copy this link",
     "COPIED_TO_CLIPBOARD": "Copied to clipboard",
-    "COPIED_TO_CLIPBOARD_DESC": "Copy this link to share via email or messages. When pasted into the search input field of the Guided Answers extension, it will navigate you straight to this content."
+    "COPIED_TO_CLIPBOARD_DESC": "Paste this code into search field on the home page to get back here.",
```

Comment:
Why do we change the text here. It is still possible to 
> Copy this link to share via email or messages.

Reply:
the text was changed to be the exact same as in Figma. also with the previous long text that spanned many lines, having the link be below that didn't look great

Reply:
I think it's best if I also message Goran about this